code/datums/gamemode/dynamic/dynamic_rulesets.dm

```diff
@@ -114,13 +116,23 @@
 	return FALSE
 
 /datum/dynamic_ruleset/proc/get_weight()
+	weight *= weight_time_day()
 	if(repeatable && weight > 1)
 		for(var/datum/dynamic_ruleset/DR in mode.executed_rules)
 			if(istype(DR,src.type))
 				weight = max(weight-2,1)
 	message_admins("[name] had [weight] weight (-[initial(weight) - weight]).")
 	return weight
 
+//Return a multiplicative weight. 1 for nothing special.
+/datum/dynamic_ruleset/proc/weight_time_day()
+	var/weigh = 1
```

Comment:
weigh**t**

Reply:
I intentionally chose to call this var weigh to not confuse with weight

Reply:
TIL weigh is a word